x/ecocredit/server/server.go

```diff
@@ -227,13 +224,25 @@ func newServer(storeKey sdk.StoreKey, paramSpace paramtypes.Subspace,
 
 	s.projectInfoTable = projectInfoTableBuilder.Build()
 
-	s.db, err = ormstore.NewStoreKeyDB(&ModuleSchema, storeKey, ormdb.ModuleDBOptions{})
+	s.db, err = ormstore.NewStoreKeyDB(&ecocredit.ModuleSchema, storeKey, ormdb.ModuleDBOptions{})
 	if err != nil {
 		panic(err)
 	}
 
 	s.basketKeeper = basket.NewKeeper(s.db, s, bankKeeper, distKeeper, storeKey)
 
+	ss, err := api.NewStateStore(s.db)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
might be nice to move the stateStore instantiation inside the `NewKeeper` functions like basket does